inception-project/inception
# #1041 - add possibility of defining keyboard shortcuts

**What's in the PR**
- First throw at key bindings for concept features (WIP)

**How to test manually**
* Open the `identifier` feature of the `Named Entity` layer in the project settings
* On the bottom of the feature editor, there are two new fields followed by a plus icon.
* Into the first field, enter a space-separated key combo following the names defined by [`wicket.contrib.input.events.key.KeyType`](https://github.com/wicketstuff/core/blob/master/input-events-parent/input-events/src/main/java/wicket/contrib/input/events/key/KeyType.java#L26-L28). The second field is actually an auto-completing concept feature editor which allows to select some item from the KB (according to the other traits settings).
* Define one or two single key shortcuts
* Go to the annotation page
* Create a new `Named Entity` annotation
* Note that the focus does **not** go into the `identifier` input field! If the focus is on a text field, the shortcuts are not considered
* Press the key for the shortcut you defined
* See that the value is being set

**Automatic testing**

**Documentation**
